jdaviz/configs/default/plugins/export/export.py

```diff
@@ -100,13 +110,24 @@ def __init__(self, *args, **kwargs):
                                                   selected='table_format_selected',
                                                   manual_options=table_format_options)
 
+        subset_format_options = ['fits', 'reg']
+        self.subset_format = SelectPluginComponent(self,
+                                                   items='subset_format_items',
+                                                   selected='subset_format_selected',
+                                                   manual_options=subset_format_options)
+
         # default selection:
         self.dataset._default_mode = 'empty'
         self.table._default_mode = 'empty'
         self.table.select_default()
         self.viewer.select_default()
         self.filename = f"{self.app.config}_export"
 
+        self.subset = SubsetSelect(self,
+                                   'subset_items',
+                                   'subset_selected',
+                                   filters=['is_spatial'])

```

Comment:
if you define this above the default selections and pass `default_mode='empty'` then we'll ensure that dataset is selected by default instead of subset (this would only be a problem though if a subset exists before instantiating the plugin)

Reply:
done